build.gradle

```diff
@@ -41,8 +41,6 @@ android {
 }
 
 dependencies {
-    api ('io.radar:sdk:2.1.9') {
-        exclude group: 'com.android.support', module: 'support-v4'
-    }
-    api 'com.android.support:appcompat-v7:28.0.0'
+    api 'io.radar:sdk:3.0.0'
+    api 'androidx.appcompat:appcompat:1.0.2'
```

Comment:
We might not need to include this if `io.radar:sdk:3.0.0` pulls it in

Reply:
Confirmed not needed.